cbrain_task/recon_all/bourreau/recon_all.rb

```diff
@@ -58,6 +58,14 @@ def setup #:nodoc:
       self.safe_symlink(cache_path, file.name)
     end
 
+    # Copy personal license for FreeSurfer
+    license = FreesurferLicense.find_all_accessible_by_user(User.find(self.user_id)).first
```

Comment:
Some of the code is unnecessary:

* Use the assoc: `find_all_accessible_by_user(self.user)`
* Use the object: `make_available(license, "license.txt")`